server/routes/api.js

```diff
@@ -1,9 +1,11 @@
 import express from "express";
-import db from "../db/connection.js"
+import db from "../db/connection.js";
+import User from "../schemas/user-schema.js";
 
 const router = express.Router();
 router.get("/hello", (_, res) => {
-  res.json({ "hello": "world" })
-})
+  res.json({ hello: "world" });
+});
 
-export default router;
\ No newline at end of file
+console.log(await db.getAll(User));
```

Comment:
I think you meant to remove this line, since this blocking call would add a lot of overhead to the router?

Reply:
Nice catch